config.json

```diff
@@ -20,6 +20,17 @@
       ]
     },
     {
+      "uuid": "dc4ad67d-0fe4-0880-5a13-3aebb8759bcc3095de1",
+      "slug": "two-fer",
+      "core": false,
```

Comment:
Yeah, this thread has really emphasized for us how important the side exercises are.

We're going to need to do actual product design work to make sure we're never blocking our power users, but even for non-power users the side exercises are going to be really central.